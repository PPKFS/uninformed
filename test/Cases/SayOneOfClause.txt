The Lab is a room.

Every turn:
	say "[one of][if the location is Lab]Lab[end if][or]blah[at random]";

Test me with "z / z / z / z".
