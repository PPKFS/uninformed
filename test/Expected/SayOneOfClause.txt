26 words
¶the¶ ¶lab¶ ¶is¶ ¶a¶ ¶room¶ ¶.¶ ¶|__¶ ¶every¶ ¶turn¶ ¶:¶ ¶say¶ ¶"[one of][if the location is Lab]Lab[end if][or]blah[at random]"¶ ¶;¶ ¶test¶ ¶me¶ ¶with¶ ¶"z / z / z / z"¶ 
17 distinct words
¶The¶ ¶the¶ 0a
¶Lab¶ ¶lab¶ 20
¶is¶ ¶is¶ 20
¶a¶ ¶a¶ 20
¶room¶ ¶room¶ 20
¶.¶ ¶.¶ 20
¶|__¶ ¶|__¶ 0a
¶Every¶ ¶every¶ 20
¶turn¶ ¶turn¶ 20
¶:¶ ¶:¶ 20
¶say¶ ¶say¶ 41
¶"[one of][if the location is Lab]Lab[end if][or]blah[at random]"¶ ¶"[one of][if the location is Lab]Lab[end if][or]blah[at random]"¶ 20
¶;¶ ¶;¶ 20
¶|__¶ ¶|__¶ 0a
¶Test¶ ¶test¶ 20
¶me¶ ¶me¶ 20
¶with¶ ¶with¶ 20
¶"z / z / z / z"¶ ¶"z / z / z / z"¶ 20
¶.¶ ¶.¶ 20
¶|__¶ ¶|__¶ 0a
¶|__¶ ¶|__¶ 0a
¶|__¶ ¶|__¶ 0a
¶|__¶ ¶|__¶ 0a
